src/votekit/graphs/pairwise_comparison_graph.py

```diff
@@ -216,3 +232,32 @@ def dominating_tiers(self) -> list[set]:
                 tier_dict[v] = {k}
         tier_list = [tier_dict[k] for k in sorted(tier_dict.keys(), reverse=True)]
         return tier_list
+
+    def has_condorcet_cycles(self) -> bool:
+        """
+        Checks if graph has any condorcet cycles, which we define as any cycle of length
+            greater than 2 in the graph.
+
+        Returns:
+            True if condorcet cycles exists, False otherwise.
+        """
+
+        if len(self.get_condorcet_cycles()) > 0:
+            return True
+
+        else:
+            return False
+
+    @cache
+    def get_condorcet_cycles(self) -> list:
+        """
+        Returns a list of condorcet cycles in the graph, which we define as any cycle of length
+            greater than 2.
+
+        Returns:
+            List of condorcet cycles
+        """
+
+        G = self.pairwise_graph
+
+        return(nx.recursive_simple_cycles(G))
```

Comment:
Do we want to return these sorted by e.g. length?

Reply:
I think that's a great idea.